internal/engine/actions/remediate/pull_request/types_yq_test.go

```diff
@@ -227,6 +232,101 @@ func TestYQExecute(t *testing.T) {
 				}
 			},
 		},
+		{
+			name: "TestTemplatedExpression",
+			yqParams: func(t *testing.T) *modificationConstructorParams {
+				t.Helper()
+
```

Comment:
Might be stretching too much here, but how about another test that has a yq expression inside the go template so both conditions are hit? (If this is too much, feel free to ignore)

Reply:
Missed that, thanks for pointing.